### PR TITLE
istio-meshery-adapter lab modified 

### LINKED ISSUE #54: Lab-Website Issue: Instructions not matching with website features
**Description**

In Step 2 of Working with Meshery and Istio lab, the user is asked to select Istio from the management menu on the Meshery site. However, at the moment there is no management menu. 

**Expected Behavior**

Lab documentation has to be updated. Istio should be selected from the Lifestyle menu. 

**Screenshots**

The lab instructions
image

The Meshery site
image

**Environment:**
 - OS: Windows
 - Browser: Brave
 - Version: Windows 11
 - Device: Laptop

---
[Optional] **To Reproduce**
1. Go to https://layer5.io/learn/service-mesh-labs/working-with-meshery-and-istio

2. Click on the Start Scenario button. 
image

3. Go to step 2. Scroll down to instructions. 

[Optional] **Additional Context**

### PR DESCRIPTION
**Pull request by KerinPithawala**
Resolves issue #54
Signed-off-by: Kerin Pithawala<kerinpithawala7@gmail.com>

**Description**
Modified instructions in step2.md file of istio-meshery-adapter as per the current Meshery UI.
Included selecting Istio from Performance menu instead of Management menu in docs because Management menu does not exist anymore.
Removed the picture of istio-adapter from assets and added updated picture.   

This PR fixes #54 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: Meshery-Adapters/istio-meshery-adapter/step2.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Meshery allows you to install any service mesh with a single click:
 
-## 1. Select `Istio` from the Management menu
+## 1. Select `Istio` from the Performance menu
```
Comment (asubedy) [inline Meshery-Adapters/istio-meshery-adapter/step2.md:3]: We find `Istio` on the `Lifecycle` Menu and not the `Performance` menu, please change this

File: Meshery-Adapters/istio-meshery-adapter/step2.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 ![Meshery adapter for Istio](./assets/istio-adapter.png)
 
-## 2. In the Istio management page:
+## 2. In the Istio Performance page:
```
Comment (asubedy) [inline Meshery-Adapters/istio-meshery-adapter/step2.md:7]: Here too, please change `Performance` to `Lifecycle`

**Review by asubedy — CHANGES_REQUESTED** [review]

**Comment by asubedy** [comment]
@KerinPithawala The picture which you changed too doesn't correctly show where `Istio` can be found. Please update the picture with a red box on `Isitio` which can be found on the `Lifecycle` Menu

File: Meshery-Adapters/istio-meshery-adapter/step2.md (hunk 1 above)
Comment (KerinPithawala) [reply Meshery-Adapters/istio-meshery-adapter/step2.md:3]: Will make the required changes

> @KerinPithawala The picture which you changed too doesn't correctly show where `Istio` can be found. Please update the picture with a red box on `Isitio` which can be found on the `Lifecycle` Menu

I will take new picture according to this review.

**Comment by KerinPithawala** [comment]
Made the required changes. Kindly review my PR.

**Comment by asubedy** [comment]
Great @KerinPithawala 👍 . Could you also please update other screenshots of the instruction. We can see the disparity between the screenshot on the instructions and the current Meshery UI.

**Comment by KerinPithawala** [comment]
Done

**Comment by asubedy** [comment]
Amazing @KerinPithawala. Just a few more Screenshots need to be updated:
- istio-sample-app.png
- bookinfo-deployed.png
- istio-sample-app.png
- bookinfo-deployed.png

**Comment by KerinPithawala** [comment]
Will do

**Comment by leecalcote** [comment]
Some good work happening here.

**Comment by KerinPithawala** [comment]
Done. Please review the same.

**Comment by asubedy** [comment]
Great @KerinPithawala. Thank you. 😊

**Review by asubedy — APPROVED** [review]

**Comment by KerinPithawala** [comment]
Please review my PR.

**Review by warunicorn19 — APPROVED** [review]
Looking good, thank you @KerinPithawala

**Pull request merged**